src/pages/Share/ShareDetailsPage.tsx

```diff
@@ -105,7 +100,7 @@ function ShareDetailsPage({
     const fileName = currentAttachment?.content.split('/').pop();
 
     const handleShare = () => {
-        if (!currentAttachment || validFilesToUpload.length === 0) {
+        if (!currentAttachment || (!!validFilesToUpload && validFilesToUpload.length === 0)) {
```

Comment:
Maybe we could use `validatedFile` here?

Reply:
I'll check it.